app/controllers/thread_controller.php

```diff
@@ -3,28 +3,42 @@ class ThreadController extends AppController
 {
     const THREADS_PERPAGE   = 10;
     const COMMENTS_PERPAGE  = 15;
+    const TRENDING_LIMIT    = 10;
 
     /*
      * Show all threads.
      */
```

Comment:
Please be consistent po in using comments. You used a single line comment po in the other parts of your code. :bow: